config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
   mount_devise_token_auth_for 'User', at: 'users'
   get '/users/me', to: 'users#me'
-  resources :users, only: %i[index show]
+  resources :users, only: %i[index show] do
+    resources :rents, only: %i[index create]
```

Comment:
In this case, I prefer to use collection instead of resources
```suggestion
    collection :rents do
      get :index
      post :create 
    end
```
You can note the difference between a resource, collection, and member [in this post](https://carlossanchezperez.wordpress.com/2013/07/21/definir-rutas-en-una-aplicacion-de-rails-con-new-resources-restful-route-customizations/)

Reply:
:raising_hand_man: My bad! You can omit my correction, and keep in this way :clap: